### ISSUE TITLE
Automated equilibration detection

### ISSUE THREAD

**Issue by MatteoA** [MobleyLab/alchemical-analysis#88]
As possible enhancement, I wonder whether it might be useful to have an option to determine the windows' equilibration period automatically as suggested by John Chodera here:
http://pubs.acs.org/doi/abs/10.1021/acs.jctc.5b00784

It could complement the time-reversed convergence plots, providing an additional estimate of the equilibrated region?

**Comment by davidlmobley** [comment]
Yes, this would be great to have. It's unlikely I'm going to be able to
have someone put this in in the short term, but I'm open to a PR which
would add this if you're interested in adding it.

Thanks.

On Thu, Apr 28, 2016 at 2:34 AM, Matteo Aldeghi <notifications@github.com>
wrote:

> As possible enhancement, I wonder whether it might be useful to have an
> option to determine the windows' equilibration period automatically as
> suggested by John Chodera here:
> http://pubs.acs.org/doi/abs/10.1021/acs.jctc.5b00784
>
> It could complement the time-reversed convergence plots, providing an
> additional estimate of the equilibrated region?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MobleyLab/alchemical-analysis/issues/88>
>



-- 
David Mobley
Associate Professor
Department of Pharmaceutical Sciences
Department of Chemistry
3134B Natural Sciences I
University of California, Irvine
Irvine, CA 92697
dmobley@uci.edu
work (949) 824-6383
cell (949) 385-2436

**Comment by MatteoA** [comment]
Hi David, sure I can try have a go at this at some point...
